### ISSUE TITLE
Archiving pages

### ISSUE THREAD

**Issue by ehbeattie** [BCcampus/bcc-sage#388]
Hi Alex,

I would like to have the ability to archive old webpages on OpenEd and BCcampus so that we can drive users to our newer and more up-to-date pages. Something like this: 
![Screen Shot 2019-08-22 at 9 52 24 AM](https://user-images.githubusercontent.com/44707412/63539189-5cc17480-c4ce-11e9-852d-45dab5aa59a2.png)

Once a page is archived, I'm wondering what the user experience would then be? Would they be redirected back to the homepage? Shown an error message? 

Cheers,

Erin

**Comment by alex-418** [comment]
@ehbeattie Yes, WordPress has some features built-in that we could use. But if I understand correctly, you'd like to un-publish older pages so that users are visiting the more recent pages instead? 

Killing the old content may have some negative effects. The reason I say this, is because the old content could be driving organic traffic from search engines that we may wish to keep. We could instead try leverage this to help them find newer content. Perhaps editing the old page with a notice there's a newer version of this content, or a more recent event? But keeping existing pages published, along with the keywords in this content does drive organic traffic to us. 

Now that being said, if we take a look at our analytics and can see the impact would be minimal, or you wish to proceed anyway, then the simplest way to achieve this would be to change the status to either "draft" or the visibility to "private" which means only users in the dashboard with editor access and above can see:

![visibility](https://user-images.githubusercontent.com/17072191/66609718-a47b8880-eb6e-11e9-8538-5bfaa99a3cc6.png)

I also think we could add our own status to the content by hooking into the right WordPress functionality that changes the status, but would prefer to avoid this as it could have unintended consequences. Such as needing to filter for this status in our theme to stop posts with this status from showing up on the front-end. 

Hope that's useful, happy to discuss further.
